[13.0][ADD] purchase_stock_picking_show_currency_rate: Show currency rate in purchase stock pickings (#1234)
OCA/purchase-workflow: purchase_stock_picking_show_currency_rate/README.rst

   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpurchase--workflow-lightgray.png?logo=github
    :target: https://github.com/OCA/purchase-workflow/tree/13.0/purchase_stock_picking_show_currency_rate
    :alt: OCA/purchase-workflow
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_stock_picking_show_currency_rate
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/142/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module shows the currency rate applied in a purchase stock picking, so that you can visually verify what is going to be applied to the valuation layer, or which one was applied once converted to company currency.

Reviewer: I don't see code for the rate once the picking is done and the valuation layer is created.

   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpurchase--workflow-lightgray.png?logo=github
    :target: https://github.com/OCA/purchase-workflow/tree/13.0/purchase_stock_picking_show_currency_rate
    :alt: OCA/purchase-workflow
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_stock_picking_show_currency_rate
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/142/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module shows the currency rate applied in purchase stock picking, so you can visually verify what is going to be applied for valuation layer, or which one was applied once converted to company currency.

Reviewer 2: ```suggestion
This module shows the currency rate applied in a purchase stock picking, so that you can visually verify what is going to be applied to the valuation layer, or which one was applied once converted to company currency.
```

**Table of contents**

.. contents::
   :local:

Configuration
=============

Enable the option for multiple currencies in your instance:

#. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies

> #. Go to any request for quotation

Reviewer: From this line to the latest point up to next paragraph, they are not configuration steps, but usage ones.

#. Change the purchase currency.
#. Add any line with storable product.
#. Confirm Order.
#. Go to Receipt.
#. The proper currency rate, based on today date and the selected currency, will be shown.

> It's really necessary to exists some rates about not company currency distinct to 1.0, for example from EUR:

Reviewer 2: ```suggestion
Rates for non-company currencies must be available and be distinct to 1.0. For example, for EUR:
```

#. Go to Invoicing > Configuration > Currencies and go to EUR
#. Go to Rates smart-button
#. Update 01/01/2010 record and change rate to 1.5

Usage
=====

To use this module, you need to:

#. Go to Purchase > Orders > Request for quotation
#. Create Purchase and set Currency distinct to company currency (EUR for example)
#. Add any line with storable product.
#. Confirm Order.
#. Go to Receipt.

> #. Rate account show according to currency defined.

Reviewer 2: ```suggestion
#. The account rate should be shown according to the currency defined previously.
```
